docs/rules/aws_acm_certificate_lifecycle.md

```diff
@@ -0,0 +1,49 @@
+# aws_acm_certificate_lifecycle
```

Comment:
Please add a link to the document index.
https://github.com/terraform-linters/tflint-ruleset-aws/blob/master/docs/rules/README.md